.gitignore

```diff
@@ -6,9 +6,6 @@ __pycache__/
 # tests
 logs/
 test_logs/
-reports/
```

Comment:
I think reports should probably stay in the gitignore. unless we want 200 csv files added to our repo from running the performance analysis over the same images over and over again 🤣